scripts/pi-hole/js/groups-common.js

```diff
@@ -36,11 +36,6 @@ function processGroupResult(data, type, done, notDone) {
   // Loop over errors and display them
   data.processed.errors.forEach(function (error) {
     console.log(error); // eslint-disable-line no-console
-    utils.showAlert(
-      "error",
-      "",
-      `Error while ${notDone} ${type} ${utils.escapeHtml(error.item)}`,
-      error.error
-    );
+    utils.showAlert("error", "", `Error while ${notDone} ${type} ${error.item}`, error.error);
```

Comment:
Why did you remove it here? This is for showing ` ${error.item}`

Reply:
Because toasts themselves ensure that *everything* they display is escaped:

![Screenshot from 2023-11-21 13-28-18](https://github.com/pi-hole/web/assets/16748619/8aaea1d4-c9a3-4d0b-9a36-038866c9ec78)

And double-escaping is not harmless, we'd get `"<"  ---escape--->  "&lt;"  ---escape--->  "&amp;lt;"`